packages/exhale/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://gitlab.com/ecodis/exhale
+TERMUX_PKG_DESCRIPTION="Open source xHE-AAC encoder"
+TERMUX_PKG_LICENSE="custom"
+TERMUX_PKG_LICENSE_FILE="include/License.htm, include/styles.css"
```

Comment:
Why is `include/style css` here?

Reply:
Tip @DLC01, you can click on resolve conversation after you have applied requested changes from GitHub UI